grapher/lineCharts/LineChart.tsx

```diff
@@ -554,7 +561,11 @@ export class LineChart
 
                             return (
                                 <circle
-                                    key={`${series.seriesName}-${series.color}`}
+                                    key={`${series.seriesName}-${
+                                        series.color
+                                    }-${
+                                        series.isProjection ? "projection" : ""
+                                    }`}
```

Comment:
Rather than appending `"projection"` multiple times, it maybe could be added to `seriesName` when the series are constructed? (Not sure if this might have some unintended effects, I _think_ not.)

https://github.com/owid/owid-grapher/blob/1c87608d84528a4788536b03039478483df4c2fb/grapher/lineCharts/LineChartUtils.ts#L13-L18